### PR TITLE
Give the ability to customize fonts base path

### PR DESCRIPTION
**Pull request by GeoSot**

### CODE REVIEW

**Comment by tsjensen** [comment]
This looks very useful! Will this be in the next release?

**Comment by WinterSilence** [comment]
need fix https://github.com/twbs/icons/blob/dc436de932cf348c4136921a089b337db8f9f041/.fantasticonrc.js#L7

File: build/font/scss.hbs [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ${{ name }}-font: "{{ name }}" !default;
-${{ name }}-font-src: {{{ fontSrc }}} !default;
+${{ name }}-font-dir: "{{ fontsUrl }}" !default;
+${{ name }}-font-file: #{${{ name }}-font-dir}/#{${{ name }}-font} !default;
+${{ name }}-font-sha: "{{ hash }}" !default;
```
Comment (XhmikosR) [inline build/font/scss.hbs:4]: It'd be better to rename `sha` to hash`

**Comment by mdo** [comment]
Moving to #1374.

**Pull request closed without merging**